### PR TITLE
TestView

### PR DESCRIPTION
**Pull request by Winternight9**
Add some tests about view function.

### CODE REVIEW

File: arbitrator/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        c = self.client
+        response = c.get(path='/profile/')
+        self.assertTemplateUsed(response, 'arbitrator/profile.html')
+
+    def test_user_can_login(self):
+        response = self.client.post('/login/',
+            {'username':self.user.username, 'password':self.userpass})
+        self.assertEqual(response.status_code, 302)
+        self.assertRedirects(response, reverse('arbitrator:test'))
+
```
Comment (KornSiwat) [inline arbitrator/tests/test_views.py]: what are those blank lines below for?

File: arbitrator/views/indexView.py [hunk 2]
```diff
@@ -7,12 +7,12 @@
 def indexView(request):
     form = LoginForm()
     context = {
-        'form':  form
+        'form':  form
```
Comment (KornSiwat) [inline arbitrator/views/indexView.py]: please remove extra space

File: arbitrator/tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        self.assertTemplateUsed(response, 'arbitrator/profile.html')
+
+    def test_user_can_login(self):
+        response = self.client.post('/login/',
+            {'username':self.user.username, 'password':self.userpass})
+        self.assertEqual(response.status_code, 302)
+        self.assertRedirects(response, reverse('arbitrator:test'))
+    
+
+    # def test_editprofile(self):
```
Comment (KornSiwat) [inline arbitrator/tests/test_views.py]: if the commented lines won't be used please delete them

**Comment by Winternight9** [comment]
@KornSiwat Already solved problems

**Pull request merged**